tests/azure/nic_test.go

```diff
@@ -0,0 +1,69 @@
+package azure_test
+
+import (
+	"fmt"
+	"math/rand"
+	"testing"
+
+	"github.com/NeowayLabs/klb/tests/lib/azure"
+	"github.com/NeowayLabs/klb/tests/lib/azure/fixture"
+)
+
+func genNicName() string {
+	return fmt.Sprintf("klb-nic-tests-%d", rand.Intn(1000))
+}
+
+func testNicCreate(t *testing.T, f fixture.F) {
+	vnet := genVnetName()
+	nsg := genNsgName()
+	nic := genNicName()
+	subnet := genSubnetName()
+	vnetAddress := "10.116.0.0/16"
+	addrnic := "10.116.1.100"
+	subnetAddress := "10.116.1.0/24"
+
+	f.Shell.Run(
+		"./testdata/create_nsg.sh",
+		nsg,
+		f.ResGroupName,
+		f.Location,
+	)
+	azure.NewNsg(f)
```

Comment:
I think github eated my comments about this :-(. These calls to New* are not required, the constructor does nothing. Why is this here ?